src/main/java/com/commercetools/sync/commons/utils/CompletableFutureUtils.java

```diff
@@ -0,0 +1,21 @@
+package com.commercetools.sync.commons.utils;
+
+import javax.annotation.Nonnull;
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+
+public class CompletableFutureUtils {
```

Comment:
pls, add private constructor to Util classes (by convention)

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/256/commits/853108b5eb23d68ca876a21358abdbc614b1f8f5